km/km_coredump.h

```diff
@@ -159,9 +163,15 @@ typedef struct km_nt_socket {
 #define KM_NT_SKSTATE_CONNECT 4
 #define KM_NT_SKSTATE_ERROR 5
 
+// Use a function so that we consistently roundup note related pieces in the rest of the code.
+static inline size_t km_nt_chunk_roundup(size_t size)
+{
+   return roundup(size, 4);
```

Comment:
I guess the fields in a note are supposed to be 4 byte aligned, but some people say they should be 8 byte aligned on a 64 bit architecture, but linux stays with 4 byte alignment on a 64 bit architecture.
So, we make sure what we add to the note will result in the following note being 4 byte aligned so that its fields will be 4 byte aligned.  The following may be helpful:
https://groups.google.com/g/generic-abi/c/vT-_QVcckXo?pli=1
It just further re-enforces the confusion for me.  So, just do what keeps them pacified.

Reply:
My point was simply that 4 seems to be a magic number. I'd like to see `const int`, with that explanation next to definition.